server/src/dbschema.js

```diff
@@ -8,6 +8,7 @@ const MAX_DB_LEVEL = 6;
 /** Create all the tables */
 exports.createTables = function () {
   console.info("Setting up tables on", db.constr);
+  console.info("Environment: ", process.env);
```

Comment:
Is there a reason for the whole environment to be in here?  Could potentially lead to leaking private keys into the logs, for instance.